ts/src/base/functions/number.ts

```diff
@@ -301,7 +324,7 @@ function omitZero (stringNumber) {
 export {
     numberToString,
     precisionFromString,
-    decimalToPrecision,
+    decimalToPrecisionWrapper as decimalToPrecision,
```

Comment:
To avoid confusions, I would rename `decimalToPrecisionWrapper` to `decimalToPrecision` and rename the original `decimalToPrecision` to `internalDecimalToPrecision`. To avoid errors in future changes